README.md

```diff
@@ -7,13 +7,196 @@ Ballerina WebSub Library
   [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/websub.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fwebsub)
   [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-websub/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-websub)
 
-The WebSub package is one of the standard library packages of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
+This library provides APIs for a WebSub Subscriber Service.
 
-This package provides the capability to easily implement a WebSub compliant **Subscriber Service**.
+[**WebSub**](https://www.w3.org/TR/websub/) is a common mechanism for communication between publishers of any kind of Web content and their subscribers, based on HTTP webhooks. Subscription requests are relayed through hubs, which validate and verify the request. Hubs then distribute new and updated content to subscribers when it becomes available. WebSub was previously known as PubSubHubbub.
 
-For more information, go to the [`websub` module](https://ballerina.io/learn/api-docs/ballerina/websub/index.html).
+[**WebSub Subscriber**](https://www.w3.org/TR/websub/#subscriber) is an implementation that discovers the `hub` and `topic URL` of a given `resource URL`, subscribes to updates at the hub, and accepts content distribution requests from the `hub`.
 
 For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/learn/by-example/).
```

Comment:
```suggestion
```

Reply:
Shall we remove this link

Reply:
Fixed with [757d14ee652ef7d7415929d028906fae3cd874e0](https://github.com/ballerina-platform/module-ballerina-websub/pull/509/commits/757d14ee652ef7d7415929d028906fae3cd874e0)